src/ontology/mondo-edit.obo

```diff
@@ -329359,7 +329333,6 @@ xref: ICD10CM:H49.4 {source="MONDO:relatedTo", source="Orphanet:254886", source=
 xref: MESH:C564926 {source="MONDO:equivalentTo"}
 xref: Orphanet:254886 {source="MONDO:equivalentTo"}
 xref: UMLS:C1850303 {source="Orphanet:254886", source="MONDO:notFoundInDiseaseSubset"}
-is_a: MONDO:0016797 {source="Orphanet:254886"} ! multiple mitochondrial DNA deletion syndrome
```

Comment:
note, according to ChatGPT, this is not a mitochondrial disease

Autosomal recessive progressive external ophthalmoplegia (arPEO) is a rare genetic disorder that primarily affects the muscles responsible for eye movement. It is characterized by weakness or paralysis of the muscles that control eye movement, leading to difficulty moving the eyes, especially in different directions. This condition can cause drooping eyelids (ptosis) and difficulty with tasks that require coordinated eye movements.

While progressive external ophthalmoplegia (PEO) is often associated with mitochondrial diseases, arPEO is not specifically classified as a mitochondrial disease. Mitochondrial diseases are a group of disorders that primarily affect the mitochondria, which are the energy-producing structures within cells. Mitochondrial diseases can affect various organs and tissues in the body, and they often involve muscle weakness and fatigue, among other symptoms.

In contrast, arPEO is typically caused by mutations in specific nuclear genes, rather than mitochondrial genes. These nuclear gene mutations can disrupt the function of the muscles involved in eye movement without necessarily affecting mitochondrial function throughout the body.

It's important to note that while arPEO is not classified as a mitochondrial disease, there can be some overlap in symptoms and clinical features between arPEO and certain mitochondrial myopathies. Genetic testing and evaluation by a healthcare professional are essential for diagnosing and distinguishing between these conditions.

Reply:
should this still have an `is_a` relation though?

Reply:
ok, resolving